en/docs/install-and-setup/setup/api-controller/managing-apis-api-products/importing-apis-via-dev-first-approach.md

```diff
@@ -34,139 +34,7 @@
-                     - GET
-                     - PUT
-                     - POST
-                     - DELETE
-                     - PATCH
-                     - OPTIONS
-                  createdTime: Dec 16, 2020 8:58:06 AM
-                  lastUpdatedTime: Dec 16, 2020 8:58:35 AM
-                  endpointConfig:
-                    endpoint_type: http
-                    sandbox_endpoints:
-                      url: https://localhost:9443/am/sample/pizzashack/v1/api/
-                    endpoint_security:
-                      production:
-                        password: admin
-                        customParameters: {}
-                        type: BASIC
-                        enabled: true
-                        username: admin
-                      sandbox:
-                        customParameters: {}
-                        enabled: false
-                    production_endpoints:
-                      url: https://localhost:9443/am/sample/pizzashack/v1/api/
-                  endpointImplementationType: ENDPOINT
-                  scopes: []
-                  operations:
-                   -
-                    id: ""
-                    target: /order
-                    verb: POST
-                    authType: Application & Application User
-                    throttlingPolicy: Unlimited
-                    scopes: []
-                    usedProductIds: []
-                   -
-                    id: ""
-                    target: /menu
-                    verb: GET
-                    authType: Application & Application User
-                    throttlingPolicy: Unlimited
-                    scopes: []
-                    usedProductIds: []
-                   -
-                    id: ""
-                    target: /order/{orderId}
-                    verb: GET
-                    authType: Application & Application User
-                    throttlingPolicy: Unlimited
-                    scopes: []
-                    usedProductIds: []
-                   -
-                    id: ""
-                    target: /order/{orderId}
-                    verb: PUT
-                    authType: Application & Application User
-                    throttlingPolicy: Unlimited
-                    scopes: []
-                    usedProductIds: []
-                   -
-                    id: ""
-                    target: /order/{orderId}
-                    verb: DELETE
-                    authType: Application & Application User
-                    throttlingPolicy: Unlimited
-                    scopes: []
-                    usedProductIds: []
-                  categories: []
-                  keyManagers:
-                   - all
-                ```
+            As an example, you can use the [Sample-Api.yaml](https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/integration/testdata/sample-api.yaml) here to generate an API Project.
```

Comment:
This is cool, using the API definition yaml that we use for testing to ensure that ii is valid and up todate.

Very innovative! We just need to make sure that we change this to point to the 4.0.0 git branch that we are going to cut at the time of the release.

Reply:
Sure. +1, will track the task to update the link after 4.0.0 release.